pingcap/dm
# *: refine dashboard

### What problem does this PR solve? 

refine the dashboard

### What is changed and how it works?

- only show instances for the selected task
- add the description for metrics
- add an overview monitoring for the task
	- see screenshot below
- add a board for the status of the instance
	- see screenshot below

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
 	- run some tasks
	- observe the dashboard

Related changes

 - Need to update the documentation
 - Need to update the `dm/dm-ansible`

![verview monitoring for the task](https://user-images.githubusercontent.com/3183039/55723625-15b17600-5a3c-11e9-8c75-3117db07c042.png)

![board for the status of the instance](https://user-images.githubusercontent.com/3183039/55774801-4390cc00-5ac9-11e9-8155-fc0ac7a8cdac.png)
